pkg/notes/cveclient.go

```diff
@@ -0,0 +1,340 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notes
```

Comment:
The notes package is already fairly large, I suggest moving it into an own package if possible.

Reply:
No problem. It's funny because I thought the opposite: I thought it was too small to merit its own package :)

I'll move it out.

Reply:
Done, I've stripped the notes pkg of all cve-related code and added a new `cve` package which has the data structures and the new client and implementation.